definitions/procedures/service/restart.rb

```diff
@@ -13,7 +13,7 @@ class Restart < Base
     PING_RETRY_INTERVAL = 30
 
     def run
-      run_service_action('stop', common_options)
+      run_service_action('stop', common_options.merge(:include_sockets => true))
       run_service_action('start', common_options)
```

Comment:
I really wonder why we don't call the restart action. I mean, `systemctl` is perfectly capable of calling stop and start itself if there's no restart defined. That also gets rid of the warning.

Reply:
@ehelms @ekohl Could you please take a look at this?